From 6ea1b4104fe2524c9c1f2cdd5f7de8c1a48f3f87 Mon Sep 17 00:00:00 2001
From: James Le Cuirot <chewi@gentoo.org>
Date: Tue, 14 Sep 2021 20:56:20 +0100
Subject: [PATCH] Fix missing dependency on configvalues.h

It's referenced by config.h, which is referenced by most things.

This was failing to build for me with Ninja on the m68k Linux platform. I don't know why it works elsewhere, could just be luck.

---
 src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 18406488dd..0e0c5f78de 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -215,6 +215,7 @@ add_library(doxymain STATIC
     ${GENERATED_SRC}/ce_parse.cpp
     # custom generated files
     ${GENERATED_SRC}/lang_cfg.h
+    ${GENERATED_SRC}/configvalues.h
     ${GENERATED_SRC}/ce_parse.h
     ${GENERATED_SRC}/resources.cpp
     #
